README.md

```diff
@@ -73,4 +73,18 @@ Ensure you have the requirements in docs installed and run the following command
 
 ```shell
 sphinx-build docs build
-```
\ No newline at end of file
+```
+
+## Running tests in docker
+
+Make sure that the containers are already running.
+
+To run all tests:
+```sh
+docker compose exec backend uv run python manage.py test
+```
+
+To run tests for a specific module:
+```sh
```

Comment:
Of you use sh here, can you switch to sh within the document?

Reply:
Ah! I'll switch it to `shell` for the new snippets since that's what was already in the doc.